Sources/StreamVideoSwiftUI/Utils/ReusePool.swift

```diff
@@ -94,4 +96,15 @@ final class ReusePool<Element: AnyObject & Hashable> {
             }
         }
     }
+
+    /// Adds a pre-created element to the available pool.
+    /// Used for prewarming to add elements beyond initial capacity.
+    ///
+    /// - Parameter element: The pre-created element to add to the pool.
+    func addToAvailable(_ element: Element) {
```

Comment:
This method defies the Pool's purpose. The pool is responsible for creating its elements on demand.